docs/tests/__snapshots__/cssvars-table.test.ts.snap

```diff
@@ -2538,6 +2538,10 @@ exports[`CSS Variables Table 1`] = `
         \\"variable\\": \\"--amplify-components-link-visited-color\\",
         \\"value\\": \\"var(--amplify-colors-font-interactive)\\"
     },
+    {
+        \\"variable\\": \\"--amplify-components-liveness-camera-module-background-color\\",
+        \\"value\\": \\"var(--amplify-colors-black)\\"
+    },
```

Comment:
Is this the only css var that developers can customize in the entire liveness module?

Reply:
Hm i'm actually not sure, @hbuchel maybe you can chime in

Reply:
Yeeeeeeahh, I think this was the start of us moving some of the inline css to tokens/classes last year but I'm not sure why we stopped there. We should definitely follow up with some more cleanup there so the tokens can be themed separately (i.e. so the border around the camera view can be themed separately from .colors.border.tertiary, etc.)

Reply:
Thanks for confirming. We should have a follow-up ticket for this.